src/components/ModelizerModelView.vue

```diff
@@ -34,42 +37,20 @@ const data = reactive({
 });
 
 /**
- * Delete a component from a tree of components
- *
- * @param {Object} component - component to find and remove.
- * @return {Boolean} true if component was found and delete otherwise false.
+ * Search a component to delete amongs all the components of all the plugins.
```

Comment:
`amongs` -> `among` or `amongst` (meaning is the same, amongst is more formal)

Reply:
Done